code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -220,7 +221,7 @@
 	light_color = LIGHT_COLOR_RED
 
 /obj/item/melee/energy/sword/pirate/New()
-	return
+	..()

```

Comment:
this shouldn't exist at all.

Reply:
Darn forgot about those. Removed in the newest commit